Remotion/Web/Development.WebTesting.IntegrationTests/BrowserLog/BrowserLogUtilityTestWithFixtureAttribute.cs

```diff
@@ -0,0 +1,128 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Linq;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using OpenQA.Selenium;
+using Remotion.Web.Development.WebTesting.BrowserLog;
+using Remotion.Web.Development.WebTesting.BrowserSession;
+using Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.BrowserLog;
+
+[TestFixture]
+[PerformBrowserLogCheck(true)]
+[BrowserLogMinimumLevel(LogLevel.Info)]
+[IgnoreBrowserLogMessage("discarded")]
+public class BrowserLogUtilityTestWithFixtureAttribute
+{
+  [Test]
+  public void IsBrowserLogCheckActive_Default ()
+  {
+    var result = BrowserLogUtility.IsBrowserLogCheckActive(new NUnitTestContext(TestContext.CurrentContext, TestExecutionContext.CurrentContext));
+    Assert.That(result, Is.EqualTo(true));
+  }
+
+  [Test]
+  [PerformBrowserLogCheck(true)]
+  public void IsBrowserLogCheckActive_EnabledOnTest ()
+  {
+    var result = BrowserLogUtility.IsBrowserLogCheckActive(new NUnitTestContext(TestContext.CurrentContext, TestExecutionContext.CurrentContext));
+    Assert.That(result, Is.EqualTo(true));
+  }
+
+  [Test]
+  [PerformBrowserLogCheck(false)]
+  public void IsBrowserLogCheckActive_DisabledOnTest ()
+  {
+    var result = BrowserLogUtility.IsBrowserLogCheckActive(new NUnitTestContext(TestContext.CurrentContext, TestExecutionContext.CurrentContext));
+    Assert.That(result, Is.EqualTo(false));
+  }
+
+  [Test]
+  [PerformBrowserLogCheck(false)]
+  [TestCase(true)]
+  [TestCase(false)]
+  public void IsBrowserLogCheckActive_DisabledOnTestSuite (bool value)
```

Comment:
```suggestion
  public void IsBrowserLogCheckActive_DisabledOnTestFixture (bool value)
```
TestSuite would be if we want to not reference the fact that it's a TestFixture-class the attribute is applied to.

Reply:
"Suite" in Nunit is a test mehtod with parameters (test cases).